runsc/sandbox/sandbox.go

```diff
@@ -1292,6 +1303,36 @@ func (s *Sandbox) createSandboxProcess(conf *config.Config, args *Args, startSyn
 	return nil
 }
 
+func SendIDToSandbox(syncFile *os.File, spec *specs.Spec) error {
+	euid := uint32(os.Geteuid())
```

Comment:
If I understand this right, the sandbox will be running under the current user. Strictly speaking, the current user might be unmapped in the container user namespace. I think sandbox/gofer processes should ideally run under the root user if that user is mapped; otherwise, they need to run under the user associated with the init process.

Reply:
I make a new PR:
https://github.com/google/gvisor/pull/11972/commits/b7994443875c8e329b95e032d0da8734b59be397